package.json

```diff
@@ -38,7 +38,7 @@
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@openzeppelin/contracts": "^4.6.0",
     "@openzeppelin/contracts-upgradeable": "^4.7.3",
-    "@venusprotocol/governance-contracts": "^2.0.0",
+    "@venusprotocol/governance-contracts": "^2.1.0-dev.1",
```

Comment:
Let's avoid `dev` versions, if it's possible

Reply:
Done https://github.com/VenusProtocol/oracle/pull/192/commits/5abef9d7e358d651a55555d2e1b28c28a3c81160